reactioncommerce/reaction
# Fix unable to open fulfill order action view

This resolves #2142 

### Steps to test
1. Sign in as admin
1. setup stripe & enable all three default Flat Rate shipping options
1. Place Order from incognito window
1. As admin, click on View All on notifications dropdown
1. Click into order from notifications panel
1. Attempt to fulfill order. 
1. Observe that action view displays correctly.
